ci/kokoro/install/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+echo "================================================================"
+printenv
+echo "================================================================"
+
+echo "================================================================"
+echo "Updating submodules."
+cd "$(dirname $0)/../../.."
```

Comment:
GIven that this is a change for the rest of the script, perhaps it should be earlier rather than within the "updating submodules" section.